components/patient-data-sharing/push-api/src/main/resources/ApplicationResources.properties

```diff
@@ -48,27 +49,27 @@ phenotips.PushPatient.male=Male
 phenotips.PushPatient.unknown=Unknown
 phenotips.PushPatient.negative=NO
 phenotips.PushPatient.oneServerMessage=There is only one configured remote server: 
-phenotips.PushPatient.patientUploadedMessage=This patient has been uploaded to this server
+phenotips.PushPatient.patientUploadedMessage=This patient was uploaded to
 phenotips.PushPatient.patientUploadedLessThanHour=less than an hour ago
 phenotips.PushPatient.patientUploadedDays=days ago
 phenotips.PushPatient.patientUploadedHours=hours ago
-phenotips.PushPatient.patientUploadedAsID=as
-phenotips.PushPatient.patientUploadedUpdate=Further pushes will attempt to update the patient on the remote server.
+phenotips.PushPatient.patientUploadedAsID=and was assigned patient ID
+phenotips.PushPatient.patientUploadedUpdate=If data in the record has changed, push to __serverID__ again to update the record there.
```

Comment:
Our prospective translators are already confused enough about what words to translate and what words not to translate. Adding yet another rule won't help. We already have a convention for parameters, we should stick to it.